docs/ossm/ossm2-migration/cluster-wide/README.md

```diff
@@ -67,6 +67,8 @@ In this procedure, we will use a proper canary upgrade with gradual migration of
 
     Here we are not using any `discoverySelectors` so the control plane will have access to all namespaces. In case you want to define `discoverySelectors`, keep in mind that all data plane namespaces you are planning to migrate from 2.6 must be matched.
 
+    > **_NOTE:_** For simplicity, we are using a minimal example for Istio resource. Read [SMCP to Istio mapping](#TODO) to see how to transform fields used in SMCP resource to fields in Istio resource.
```

Comment:
This is good but somewhere we also need to tell users how to setup their `Istio` resource with `extensionProviders`. The 2.6 pre-migration checklist ends with these fields being set on the `SMCP` and if someone is just blindly following the examples it is easy to miss that you need to set them on the `Istio` resource as well.